Niraj-1729/Hacktoberfest
# Add - Dice Roll Game From HTML, CSS, JavaScript

I am creating the Dice Roll Game by using HTML, CSS, and JavaScript.

Add #40 

Screenshot - 

https://github.com/Niraj-1729/Hacktoberfest/assets/91898349/8d8cd5a4-16b6-4e06-9964-a1539b849c0b

How should this be tested?
1. Go to the Hacktoberfest folder.
2. Open the folder Dice-Roll.
3. Then see the Project.
